project.yml

```diff
@@ -23,7 +23,7 @@ packages:
     majorVersion: 0.18.0
   Nimble:
     url: https://github.com/Quick/Nimble/
-    majorVersion: 11.0.0
+    majorVersion: 10.0.0
```

Comment:
Change was necessary because of error:

`Compiling for iOS 12.1, but module 'Nimble' has a minimum deployment target of iOS 13.0:`